@cursor fix failing workflows:
```
Run npm ci
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: mcp-router@0.1.0
npm error Found: @typescript-eslint/eslint-plugin@5.62.0
npm error node_modules/@typescript-eslint/eslint-plugin
npm error   dev @typescript-eslint/eslint-plugin@"^8.26.0" from the root project
npm error   @typescript-eslint/eslint-plugin@"^5.5.0" from eslint-config-react-app@7.0.1
npm error   node_modules/eslint-config-react-app
npm error     eslint-config-react-app@"^7.0.1" from react-scripts@5.0.1
npm error     node_modules/react-scripts
npm error       peer react-scripts@">=2.1.3" from react-app-rewired@2.2.1
npm error       node_modules/react-app-rewired
npm error         dev react-app-rewired@"^2.2.1" from @mcp-router/frontend@1.0.0
npm error         packages/frontend
npm error       1 more (@mcp-router/frontend)
npm error   1 more (eslint-plugin-jest)
npm error
npm error Could not resolve dependency:
npm error dev @typescript-eslint/eslint-plugin@"^8.26.0" from the root project
npm error
npm error Conflicting peer dependency: @typescript-eslint/parser@8.44.1
npm error node_modules/@typescript-eslint/parser
npm error   peer @typescript-eslint/parser@"^8.44.1" from @typescript-eslint/eslint-plugin@8.44.1
npm error   node_modules/@typescript-eslint/eslint-plugin
npm error     dev @typescript-eslint/eslint-plugin@"^8.26.0" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /home/runner/.npm/_logs/2025-09-29T12_14_48_485Z-eresolve-report.txt
npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2025-09-29T12_14_48_485Z-debug-0.log
Error: Process completed with exit code 1.
```

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,1 @@
+legacy-peer-deps=true

--- a/package.json
+++ b/package.json
@@ -43,11 +43,11 @@
   },
   "devDependencies": {
     "@svgr/webpack": "^8.1.0",
-    "@typescript-eslint/eslint-plugin": "^8.26.0",
-    "@typescript-eslint/parser": "^8.26.0",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
     "concurrently": "^8.2.2",
     "css-select": "^5.1.0",
-    "eslint": "^9.21.0",
+    "eslint": "^8.57.0",
     "eslint-config-prettier": "^10.0.2",
     "eslint-plugin-markdown": "^5.1.0",
     "eslint-plugin-prettier": "^5.2.3",
@@ -62,6 +62,11 @@
     "typescript": "^4.9.5",
     "workbox-build": "^7.3.0",
     "workbox-webpack-plugin": "^7.3.0"
+  },
+  "overrides": {
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
+    "eslint": "^8.57.0"
   },
   "dependencies": {
     "@types/uuid": "^10.0.0",

--- a/packages/frontend/package.json
+++ b/packages/frontend/package.json
@@ -33,13 +33,13 @@
     "@types/node": "^20.10.0",
     "@types/react": "^18.2.39",
     "@types/react-dom": "^18.2.17",
-    "@typescript-eslint/eslint-plugin": "^6.12.0",
-    "@typescript-eslint/parser": "^6.12.0",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
     "assert": "^2.1.0",
     "browserify-zlib": "^0.2.0",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
-    "eslint": "^8.54.0",
+    "eslint": "^8.57.0",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "https-browserify": "^1.0.0",
@@ -56,6 +56,11 @@
     "typescript": "^5.3.2",
     "url": "^0.11.3",
     "util": "^0.12.5"
+  },
+  "overrides": {
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
+    "eslint": "^8.57.0"
   },
   "browserslist": {
     "production": [

--- a/packages/shared/package.json
+++ b/packages/shared/package.json
@@ -26,11 +26,16 @@
   "dependencies": {},
   "devDependencies": {
     "@types/node": "^20.10.0",
-    "@typescript-eslint/eslint-plugin": "^6.12.0",
-    "@typescript-eslint/parser": "^6.12.0",
-    "eslint": "^8.54.0",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
+    "eslint": "^8.57.0",
     "rimraf": "^5.0.5",
     "typescript": "^5.3.2"
+  },
+  "overrides": {
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
+    "eslint": "^8.57.0"
   },
   "files": [
     "dist",

